--- origsrc/libgnomedb-3.99.3/extra/Makefile.am	2008-05-18 14:04:17.000000000 -0500
+++ src/libgnomedb-3.99.3/extra/Makefile.am	2008-09-03 03:39:52.916736300 -0500
@@ -17,13 +17,13 @@
         -DDATADIR=\""$(datadir)"\" \
         -DLIBDIR=\""$(libdir)"\"
 
-bin_PROGRAMS = gnome-db-browser
+bin_PROGRAMS = gnome-db-browser-4.0
 
-gnome_db_browser_SOURCES = \
+gnome_db_browser_4_0_SOURCES = \
 	gnome-db-browser.c
 
-gnome_db_browser_LDFLAGS = $(EXTRALDFLAGS)
-gnome_db_browser_LDADD = \
+gnome_db_browser_4_0_LDFLAGS = $(EXTRALDFLAGS)
+gnome_db_browser_4_0_LDADD = \
 	$(top_builddir)/libgnomedb/libgnomedb-4.0.la $(LIBGNOMEDB_LIBS) $(LIBGLADE_LIBS) \
 	$(top_builddir)/libgnomedb-extra/libgnomedb_extra-4.0.la
 
--- origsrc/libgnomedb-3.99.3/extra/demos/Makefile.am	2008-05-18 14:04:15.000000000 -0500
+++ src/libgnomedb-3.99.3/extra/demos/Makefile.am	2008-09-03 03:40:22.949331700 -0500
@@ -47,22 +47,22 @@
         -DDATADIR=\""$(datadir)"\" \
         -DLIBDIR=\""$(libdir)"\"
 
-bin_PROGRAMS = gnome-db-demo
+bin_PROGRAMS = gnome-db-demo-4.0
 
 BUILT_SOURCES = demos.h
 
 demos.h: $(demos) geninclude.pl
 	(here=`pwd` ; cd $(srcdir) && $(PERL) $$here/geninclude.pl $(demos)) > demos.h
 
-gnome_db_demo_SOURCES = 		\
+gnome_db_demo_4_0_SOURCES = 		\
 	$(demos)		\
 	demo-common.h		\
 	main.c			\
 	demos.h
 
-gnome_db_demo_DEPENDENCIES = $(DEPS)
-gnome_db_demo_LDFLAGS =  $(EXTRALDFLAGS)
-gnome_db_demo_LDADD =  \
+gnome_db_demo_4_0_DEPENDENCIES = $(DEPS)
+gnome_db_demo_4_0_LDFLAGS =  $(EXTRALDFLAGS)
+gnome_db_demo_4_0_LDADD =  \
 	$(top_builddir)/libgnomedb/libgnomedb-4.0.la $(LIBGNOMEDB_LIBS) $(LIBGLADE_LIBS) \
 	$(top_builddir)/libgnomedb-extra/libgnomedb_extra-4.0.la \
         $(graphlib)
--- origsrc/libgnomedb-3.99.3/glade3/gnomedb.xml.in	2008-05-18 14:04:37.000000000 -0500
+++ src/libgnomedb-3.99.3/glade3/gnomedb.xml.in	2008-09-03 09:15:50.237056000 -0500
@@ -1,4 +1,4 @@
-<glade-catalog name="gnomedb" library="gladegnomedb" depends="gtk+" domain="glade-3" book="libgnomedb">
+<glade-catalog name="gnomedb" library="gladegnomedb" depends="gtk+" domain="glade-3" book="libgnomedb-4.0">
   <init-function>glade_gnome_db_init</init-function>
   <glade-widget-classes>
     <glade-widget-class name="GnomeDbBasicForm" generic-name="gnome-db-basic-form" title="Basic form (for data in a GdaSet)">
